### DISCUSSION TITLE
Blob as string

### DISCUSSION THREAD

**Discussion by marcin-bury** [asfernandes/node-firebird-drivers#96] [Q&A] [answered]
Hello

First let me ask if there is another place I could ask questions regardnig the usage of this package instead of creating new issue (which may be annoying for the author(s) ).

If not, let me ask it here: is there a way to automatically convert text blobs to strings (while fetching in resultset) or vice versa (string to blob) while performing insert or update using params array?

TIA
Marcin

**Comment by asfernandes** [comment, marked as answer]
> Hello
> 
> First let me ask if there is another place I could ask questions regardnig the usage of this package instead of creating new issue (which may be annoying for the author(s) ).
> 

Discussions, and I've moved to there.


> If not, let me ask it here: is there a way to automatically convert text blobs to strings (while fetching in resultset) or vice versa (string to blob) while performing insert or update using params array?
> 

Only if you use CAST in the SQL statements.
